devpi_ldap/main.py

```diff
@@ -175,7 +175,7 @@ def _search(self, conn, config, **kw):
             if any(attribute_name in x.get('attributes', {}) for x in conn.response):
                 def extract_search(s):
                     if 'attributes' in s:
-                        return s['attributes'][attribute_name]
+                        return [s['attributes'][attribute_name]]
```

Comment:
So what is the solution? Detect lists and return them as is and wrap everything else in a list?